packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -159,7 +159,7 @@ export class Relayer {
 
       // Either this bot is the slow relayer for this relay OR the relay is past 15 mins and is settleable by anyone.
       const settleableRelays = this.l1Client
-        .getSettleableRelayedDepositsForL1Token(l1Token)
+        .getSettleableRelayedDepositsForL1Token(l1Token, this.l2Client.chainId)
```

Comment:
Why not filter on `chainId` in this file instead of updating the `InsuredBridgeL1Client. getSettleableRelayedDepositsForL1Token` function? I think its a bit "safer" since we're not updating the upstream code

Reply:
sure. please take a look.